src/transaction.ts

```diff
@@ -359,6 +359,11 @@ export class Snapshot extends EventEmitter {
     const callback =
       typeof gaxOptionsOrCallback === 'function' ? gaxOptionsOrCallback : cb!;
 
+    if (!this.session) {
+      throw new GoogleError(
+        'Transaction has been closed as it was running for more than 60 minutes'
```

Comment:
Should we add this error message `Transaction has been closed as it was running for more than 60 minutes` in common file and use it at all the places. This would avoid change at multiple places if we have to change it later